src/static/css/pages/bureau.less

```diff
@@ -6,33 +6,21 @@
 // Size of play button image;
 @play-btn-size__px: 73px;
 
-.bureau-mission {
-    margin-bottom: unit(45px / @base-font-size-px, em);
+// Override general slick carousel overrides defined in mobile-carousel.less.
+// Removes bottom border, because the-bureau has a bordered block below
+// its carousel.
+.the-bureau .js-mobile-carousel.slick-slider {
```

Comment:
The Slick carousel _could_ be used on the bureau page outside of the .js-mobile-carousel class. We shouldn't do that, but this protects it from breaking were someone (_not on this team, of course_) to do that.